### PR TITLE
added py file

### PR DESCRIPTION
**Pull request by Surinder07**
Added some py file

### CODE REVIEW

**Review by meghladumor — COMMENTED** [review]
There are some issues which i can see yes!

**Review by kaurraman275 — COMMENTED** [review]
Please correct psvm.

**Review by payalarora1827 — COMMENTED** [review]
Can you print some message!

File: index.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+public class testCode {
+    public stati main(Strg[] args) {
```
Comment (OliGanguly) [inline index.py:2]: public static void main(String []args){}

Comment (chiragchadha11) [inline index.py:2]: please check the syntax of main() method

Comment (kaurraman275) [inline index.py:2]: Please check psvm

**Review by chiragchadha11 — CHANGES_REQUESTED** [review]

**Review by OliGanguly — CHANGES_REQUESTED** [review]

File: index.py (hunk 1 above)
Comment (meghladumor) [inline index.py:2]: its should be public static void main(String[] args)

File: index.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+public class testCode {
```
Comment (kaurraman275) [inline index.py:1]: Please write class name with capital letter

Comment (payalarora1827) [inline index.py:1]: Class name should be capital - please rename to TestCode

**Comment by Surinder07** [comment]
> Can you print some message!

okay

**Pull request closed without merging**